base-theme/layouts/partials/include_js.html

```diff
@@ -1,11 +1,7 @@
 {{ $context := .context }}
-{{ $url := .url }}
-{{ $type := (printf "%T" $url) }}
-{{ if eq $type "[]interface {}" }}
-{{ with index $url 0 }}
-<script src="{{ partial "webpack_url.html" (dict "context" $context "url" .) }}"></script>
-{{ end }}
```

Comment:
This conditional was added in https://github.com/mitodl/ocw-hugo-themes/pull/807 to deal with a local dev problem. It seems to no longer be an issue.